app/src/main/res/values/strings.xml

```diff
@@ -207,4 +207,7 @@
   <string name="generic_dismiss">Dismiss</string>
   <string name="rating_title">Excellent</string>
   <string name="rating_subtitle">1532 reviews</string>
+  <string name="generic_range">Range</string>
```

Comment:
small naming nitpick: these seem specific to the slider component, rather than generic, so maybe pre-fix them with `slider` rather than `generic`?